inc/dbconnection.class.php

```diff
@@ -470,4 +470,4 @@ static function changeCronTaskStatus($enable=true) {
    }
 
 }
-?>
\ No newline at end of file
+?>
```

Comment:
@trasher bad closing line can raise awfull issues... (because of unwanted output, when sending doc to browser).